src/ng-package.schema.json

```diff
@@ -55,29 +55,12 @@
           "type": "string",
           "default": ""
         },
-        "embedded": {
-          "description": "An array of external dependencies that will be embedded in the final bundle.",
-          "type": "array",
-          "items": {
-            "type": "string"
-          }
-        },
         "umdModuleIds": {
           "description":
             "A map of external dependencies and their correspondent UMD module identifiers. Map keys are TypeScript / EcmaScript module identifiers. Map values are UMD module ids. The purpose of this map is to correctly bundle an UMD module file (with `rollup`). By default, `rxjs`, `tslib` and `@angular/*` dependency symbols are supported.",
           "type": "object",
           "additionalProperties": true
         },
-        "comments": {
-          "description": "Use \"all\" to keep all, \"none\" to remove all the comments or regexes",
-          "type": "string",
-          "default": "all"
-        },
-        "licensePath": {
-          "description": "Licesnse file path to prepend to the bundles",
-          "type": "string",
-          "default": ""
-        },
```

Comment:
Note: all three breaking changes are good imo. It's required for moving to APF 6 now. The functionality has to be restored later in v3. v2 will continue to keep that features.